### PR TITLE
Update install notes

### PR DESCRIPTION
**Pull request by sylvchev**
Fix installation notes for documentation and Readmes pages on GH.

### CODE REVIEW

**Comment by sylvchev** [comment]
@bruAristimunha We will need to update CONTRIBUTING page when pre-commit will be integrated from #502

File: README.rst [hunk 1]
```diff
@@ -33,31 +33,19 @@ Installation Braindecode
 
 1. Install pytorch from http://pytorch.org/ (you don't need to install torchvision).
 
-2. Install `MOABB <https://github.com/NeuroTechX/moabb>`_ via pip (needed if you want to use MOABB datasets utilities):
+1. If you want to download EEG datasets from `MOABB <https://github.com/NeuroTechX/moabb>`_, install it:
```
Comment (sliwy) [inline README.rst:36]: the numbering is not working here and in the next point

File: docs/install/install_pip.rst [hunk 2]
```diff
@@ -14,13 +14,11 @@ Below are the installation commands for the most common use cases.
 
    pip install braindecode
 
-Braindecode can also be installed with sets of optional dependencies, for example to import datasets from MOABB:
+Braindecode can also be installed alon MOABB to download open datasets:
```
Comment (sliwy) [inline docs/install/install_pip.rst:17]:
```suggestion
Braindecode can also be installed along MOABB to download open datasets:
```

**Review by sliwy — APPROVED** [review]

**Comment by sylvchev** [comment]
Thanks @sliwy

**Pull request merged**